src/components/LoginPage.js

```diff
@@ -220,43 +264,20 @@ const LoginPage = ({ theme }) => {
           !formData.base_url.match(/^(http|https):\/\/[a-zA-Z0-9\-.]+$/)
         )
           return;
-        const versionUrl = `${formData.base_url}/_synapse/admin/v1/server_version`;
-        fetchUtils
-          .fetchJson(versionUrl, { method: "GET" })
-          .then(({ json }) => {
-            setServerVersion(
-              `${translate("synapseadmin.auth.server_version")} ${
-                json["server_version"]
-              }`
-            );
-          })
-          .catch(_ => {
-            setServerVersion("");
-          });
-
-        // Set SSO Url
-        const authMethodUrl = `${formData.base_url}/_matrix/client/r0/login`;
-        let supportPass = false,
-          supportSSO = false;
+        const wellKnownUrl = `${formData.base_url}/.well-known/matrix/client`;
```

Comment:
Duplicate code: Well-Known URL is already defined in line 245. Please extract that into a function that can be used by both.